### PR TITLE
Stop using .container-fluid and .breadcrumb bootstrap classes

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Comment by KKoukiou** [comment]
@marusak @martinpitt can any of you review this one? It's an easy one

**Comment by marusak** [comment]
I'll take this one.

**Comment by marusak** [comment]
Looking at the UI first.
- in graphs there are two issues - There is no `>` between `Overview` and the subsequent item, and the other `>` is not spaced properly.
broken

- it feels misaligned on services page: (the same with applications, hardware details, virtual machines)
servicesmissaligned
(at least when compared to logs (networking, storage...) (plus the `>` is bold on those pages)
logscompared

- the spacing on accounts feels wrong (at least not consistent with the rest of the cockpit)
acountspacing

**Comment by KKoukiou** [comment]
@marusak I think both issues you mentioned are issues in master and are not caused by this PR.

**Comment by KKoukiou** [comment]
I will address the issues you mentioned and repost.

**Comment by KKoukiou** [comment]
@marusak I fixed both issues you found in another PR, which is already merged, and rebased. https://github.com/cockpit-project/cockpit/pull/14550

Can you please re-review?

**Review by marusak — CHANGES_REQUESTED** [review]
One regression unfortunately.

File: pkg/kdump/kdump-view.jsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ import cockpit from "cockpit";
 
 import React from "react";
 import { OnOffSwitch } from "cockpit-components-onoff.jsx";
-import { Button, Tooltip, TooltipPosition } from "@patternfly/react-core";
+import { Button, Tooltip, TooltipPosition, Page, PageSection, PageSectionVariants } from "@patternfly/react-core";
```
[the remark this answers is not in the document]
Comment (KKoukiou) [reply pkg/kdump/kdump-view.jsx:24]: Good catch, reposting.

[the remark this answers is not in the document]
Comment (KKoukiou) [reply pkg/kdump/kdump-view.jsx]: btw I just noticed that this page is broken on mobile, even on master, it does not wrap at all. I will send another PR for fixing this.

**Review by marusak — APPROVED** [review]
Awesome, thanks!

**Pull request merged**